arbnode/node.go

```diff
@@ -263,38 +270,6 @@ func DeployOnL1(ctx context.Context, l1client arbutil.L1Interface, deployAuth *b
 		return nil, fmt.Errorf("error parsing rollup created log: %w", err)
 	}
 
-	sequencerInbox, err := bridgegen.NewSequencerInbox(info.SequencerInbox, l1client)
-	if err != nil {
-		return nil, fmt.Errorf("error getting sequencer inbox: %w", err)
-	}
-
-	// if a zero sequencer address is specified, don't authorize any sequencers
-	if sequencer != (common.Address{}) {
-		tx, err = sequencerInbox.SetIsBatchPoster(deployAuth, sequencer, true)
-		err = andTxSucceeded(ctx, l1Reader, tx, err)
-		if err != nil {
-			return nil, fmt.Errorf("error setting is batch poster: %w", err)
-		}
-	}
```

Comment:
Confirmed this logic has been moved into the contract https://github.com/OffchainLabs/nitro-contracts/blame/252e690df81dc6555e036c9b1f76a374721bcfd6/src/rollup/RollupCreator.sol#L86